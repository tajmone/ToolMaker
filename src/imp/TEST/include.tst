%%SET GramName(M2)
%%SET SemName("%%(GramName).SEM")
%%(SemName)
%%INCLUDE("%%(GramName).SEM")
%%INCLUDE("%%(SemName)")
