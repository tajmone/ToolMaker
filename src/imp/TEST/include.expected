########## include ##########
M2.SEM
Detta {r M2.SEM!
Detta {r M2.SEM!
